libsqlite3-sys/build.rs

```diff
@@ -111,7 +129,20 @@ mod build {
 
     pub fn main(_out_dir: &str, out_path: &Path) {
         let header = find_sqlite();
-        bindings::write_to_out_dir(header, out_path);
+        if cfg!(feature = "bundled") && !cfg!(feature = "buildtime_bindgen") {
+            // We can only get here if `bundled` and `sqlcipher` were both
+            // specified (and `builtime_bindgen` was not). In order to keep
+            // `rusqlite` relatively clean we hide the fact that `bundled` can
+            // be ignored in some cases, and just use the bundled bindings, even
+            // though the library we found might not match their version.
+            // Ideally we'd perform a version check here, but doing so is rather
+            // tricky, since we might not have access to executables (and
+            // moreover, we might be cross compiling).
+            std::fs::copy("sqlite3/bindgen_bundled_version.rs", out_path)
```

Comment:
Here, `sqlite3/bindgen_bundled_version.rs` is your custom sqlcipher bindgen ?

Reply:
It's the same as the normal `bindgen_bundled_version.rs` present in the repo. It's not exactly ideal to use it in this case, but for us we always know we're using a recent sqlcipher, and in any case where we're not, it's not really that much worse than using a more recent header file with an older sqlite. We emit a warning if you configure both bundled and sqlcipher for this reason, however.

Reply:
Ok!